test/recipes/95-test_external_pyca_data/cryptography.sh

```diff
@@ -44,10 +44,39 @@ pip install -U pip
 
 cd pyca-cryptography
 
+# Handling Rust
+export RUSTUP_HOME="$PWD/.cache/.rustup"
+export CARGO_HOME="$PWD/.cache/.cargo"
+export PATH="$CARGO_HOME/bin:$PATH"
+RUST_VERSION="1.85.1"
+
+# Install Rust 
```

Comment:
please remove the trailing whitespace on this line L53 ("Install Rust "), per the OpenSSL coding guidelines. Thanks